ThumbnailItem/ThumbnailItem.module.less

```diff
@@ -44,23 +45,7 @@
 
 		.content {
 			padding: 0 @agate-component-spacing;
-			font-size: @agate-thumbnailitem-content-font-size
-		}
-
-	  	.focus({
-			&.item {
-				.label {
-					color: @agate-thumbnailitem-label-focus-text-color;
-				}
-			}
-		});
-
-		&.selected {
-			&.item {
-				.label {
-					color: @agate-thumbnailitem-label-selected-text-color;
-				}
-			}
+			font-size: @agate-thumbnailitem-content-font-size;
```

Comment:
Is this needed? Also GUI says `32px`
![font-size](https://user-images.githubusercontent.com/63335823/114153906-82f88a00-9928-11eb-8dec-7ae187e0723e.PNG)